lib/mods/replace-methods.js

```diff
@@ -30,17 +38,26 @@ function replaceGlobalMethodCalls(node, imports) {
   if (nodeTypes.isPropertyAccessExpression(node) &&
       nodeTypes.isCallExpression(node.getParent()) &&
       isGlobal(node.getExpression())) {
-    const callerType = node.getExpression().getType().getSymbol().getName();
+    const baseType = node.getExpression().getType();
+    let callerType = baseType.getSymbol().getName();
```

Comment:
I think callerType should be called baseTypeName.

Reply:
You could also move this down to after the `if`, and change the `callerType = ` to be `baseType =` and then lose the `.getSymbol().getName()`.